frontend/src/features/dsPreview/DatasetPreviewPage.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import useDimensions from 'react-use-dimensions'
+
+import { selectDataset, selectIsDatasetLoading } from '../dataset/state/datasetState'
+import DatasetComponent from '../dsComponents/DatasetComponent'
+import Spinner from '../../chrome/Spinner'
+import ContentBox from '../../chrome/ContentBox'
+import Icon from '../../chrome/Icon'
+import ContentBoxTitle from '../../chrome/ContentBoxTitle'
+import DownloadDatasetButton from '../download/DownloadDatasetButton'
+import RelativeTimestampWithIcon from '../../chrome/RelativeTimestampWithIcon'
+import UsernameWithIcon from '../../chrome/UsernameWithIcon'
+import BodyPreview from '../dsComponents/body/BodyPreview'
+import DatasetHeader from '../dataset/DatasetHeader'
+import NavBar from '../navbar/NavBar'
+import DatasetNavSidebar from '../dataset/DatasetNavSidebar'
+import DeployingScreen from '../deploy/DeployingScreen'
+
+import { selectSessionUserCanEditDataset } from '../dataset/state/datasetState';
+import { QriRef } from '../../qri/ref'
+
+interface DatasetPreviewPageProps {
+  qriRef: QriRef
+}
+
+const DatasetPreviewPage: React.FC<DatasetPreviewPageProps> = ({
+  qriRef
+}) => {
+  const dataset = useSelector(selectDataset)
+  const loading = useSelector(selectIsDatasetLoading)
+  const editable = useSelector(selectSessionUserCanEditDataset)
+
+  const [versionInfoContainer, { height: versionInfoContainerHeight }] = useDimensions();
+  const [expandReadme, setExpandReadme] = useState(false)
+
+  let readmeContainerHeight = versionInfoContainerHeight || 'auto'
+
+  if (expandReadme) {
+    readmeContainerHeight = 'auto'
+  }
+
+  return (
+    <div className='flex flex-col h-full w-full' style={{ backgroundColor: '#f3f4f6'}}>
```

Comment:
pull this color into a constants file: `/frontend/src/colors.ts` & name it. If we haven't given it a name (like, say "qri blue"), name the color according to its human name from https://coolors.co. In this case `fef4f6` is named `cultured`.

```tsx

export default const colors = {
  cultured: '#f3f4f6'
}
```

Import & use that constant here instead of directly referencing it. Apply this to all hardcoded colors as you come across while working on the codebase

Reply:
It's actually `qrigray-100`... fixed!